src/package.json

```diff
@@ -40,6 +40,13 @@
   "devDependencies": {
     "tns-core-modules": "^3.0.0 || ^3.0.0-rc.1",
```

Comment:
Ok, I will fix it later; but it should be like that so it could fit with newer nativescript releases